aspnetcore/blazor/components/integration.md

```diff
@@ -22,24 +22,171 @@ Razor components can be integrated into Razor Pages, MVC, and other types of ASP
 
 Use the guidance in the following sections depending on the project's requirements:
 
-<!-- * Blazor support can be [added to an ASP.NET Core app](#add-blazor-support-to-an-aspnet-core-app). -->
+* Blazor support can be [added to an ASP.NET Core app](#add-blazor-support-to-an-aspnet-core-app).
 * For interactive components that aren't directly routable from user requests, see the [Use non-routable components in pages or views](#use-non-routable-components-in-pages-or-views) section. Follow this guidance when the app embeds components into existing pages and views with the [Component Tag Helper](xref:mvc/views/tag-helpers/builtin-th/component-tag-helper).
 * For interactive components that are directly routable from user requests, see the [Use routable components](#use-routable-components) section. Follow this guidance when visitors should be able to make an HTTP request in their browser for a component with an [`@page`](xref:mvc/views/razor#page) directive.
 
-<!-- HOLD
-
 ## Add Blazor support to an ASP.NET Core app
 
-This section covers adding Blazor support to an ASP.NET Core app.
+This section covers adding Blazor support to an ASP.NET Core app:
+
+* [Add static server Razor component rendering](#add-static-server-razor-component-rendering)
+* [Enable interactive server rendering](#enable-interactive-server-rendering)
+* [Enable interactive Auto or WebAssembly rendering](#enable-interactive-auto-or-webassembly-rendering)
 
 > [!NOTE]
 > For the examples in this section, the example app's name and namespace is `AspNetCoreApp`.
 
+### Add static server Razor component rendering
+
+Add the following assets to the app, updating the `{APP NAMESPACE}` to the app's namespace as each file is added.
+
+`Components/_Imports.razor`:
+
+```razor
+@using System.Net.Http
+@using System.Net.Http.Json
+@using Microsoft.AspNetCore.Components.Forms
+@using Microsoft.AspNetCore.Components.Routing
+@using Microsoft.AspNetCore.Components.Web
+@using Microsoft.AspNetCore.Components.Web.Virtualization
+@using Microsoft.JSInterop
+@using {APP NAMESPACE}
+@using {APP NAMESPACE}.Components
+```
+
+`Components/Routes.razor`:
+
+```razor
+<Router AppAssembly="@typeof(Program).Assembly">
+    <Found Context="routeData">
+        <RouteView RouteData="@routeData" />
```

Comment:
Can we add a note about how you might want to add a default layout here that matches the rest of your app and link to the related docs on Blazor layouts?